src/libostree/ostree-repo-prune.c

```diff
@@ -189,6 +189,44 @@ _ostree_repo_prune_tmp (OstreeRepo *self,
   return TRUE;
 }
 
+static gboolean
+get_objects_from_refs_only (OstreeRepo   *repo,
+                            OstreeRepoCommitTraverseFlags flags,
+                            GHashTable   **objects,
```

Comment:
This is OK as is, but I did change the new internal API https://github.com/ostreedev/ostree/pull/2635/files#diff-140b1d44c51cd554eceeef9700a401e26e72f310df9694d61c655b5d6f089e47R4949 to directly return the `GHashTable` which I think is cleaner.

Reply:
I agree, thats a lot cleaner. I'll make that change :+1: